kubos/test/test_target.py

```diff
@@ -0,0 +1,146 @@
+# Kubos CLI
+# Copyright (C) 2017 Kubos Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from mock import patch, MagicMock, call, mock_open
+from pip.utils import get_installed_version
+
+from kubos import target
+from kubos.utils.constants import *
+
+test_global_target = 'test-global-target'
+class Args():
+    def __init__(self):
+        self.set_target = None
+        self.target = None
+        self.list = None
+        self.plain = []
+
+class TargetTest(unittest.TestCase):
+    @patch('kubos.target.load_target_list')
+    @patch('kubos.target.get_project_type')
+    def test_add_options(self, get_project_type, load_target_list):
```

Comment:
Totally makes sense. These tests seem based on strings too much. I added it for completeness, but I will remove it. 👍